lib/api/request_models/login_request.dart

```diff
@@ -3,11 +3,13 @@ import 'package:client_common/api/request_models/api_request.dart';
 class LoginRequest extends ApiRequest {
   final String email;
   final String password;
+  final bool keep;
 
-  LoginRequest(this.email, this.password);
+  LoginRequest(this.email, this.password, this.keep);
 
   Map<String, String> toJson() => {
```

Comment:
you could use a Map<String, dynamic> to avoid having to call .toString() on keep